personalwebsite/aboutme/templates/aboutme/about_me_page_view.html

```diff
@@ -1,110 +1,211 @@
-                    the
-                    md breakpoint).</h3>
-            </div>
+        <div class="col-md-12 col-lg-6 col-xl-6">
+            <img class="rounded img-fluid my-image mx-auto d-block" src="{% static 'images/placeholder.png' %}"
+                alt="Empty">
+        </div>
+
+        <div class="col-md-12 col-lg-6 col-xl-6">
+            <h2 class="text-left my-introduction-title font-bold">Lorem Ipsum passage</h2>
+            <p class="text-left my-introduction-paragraph font-medium" style="font-size: 1.1rem;">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua.</p>
+            <p class="text-left my-introduction-paragraph font-medium" style="font-size: 1.1rem;">
+                Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+                consequat.</p>
+            <p class="text-left my-introduction-paragraph font-medium" style="font-size: 1.1rem;">
+                Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+            </p>
+            <p class="text-left my-introduction-paragraph font-medium" style="font-size: 1.1rem;">
+                Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
+                laborum.
+            </p>
         </div>
+
     </div>
 </div>
 
 {% endif %}
 
-
 {% if sections %}
 
-    {% for section in sections %}
-    <div class="my-group">
-        <div class="page-title border-bottom">
-            <h1 class="display-4 font-family-roboto font-medium">{{ section.title }}</h1>
-        </div>
+{% for section in sections %}
+<div class="my-group">
+    <div class="page-title">
+        <h2 class="my-title font-medium">{{ section.title }}</h2>
+    </div>
 
-        {% for content in contents %}
-            {% if content.section.title == section.title %}
-            <div class="row row-cols-1 row-cols-md-3 mt-4">
-                <div class="col mb-4">
-                    <div class="card">
-                        {% if content.image %}
-                            <div class="image-div">
-                                <img src="{{ content.image.url }}" class="card-img-top rounded my-image"
-                                alt="Responsive image">
-                            </div>
-                        {% endif %}
-                        <div class="card-body">
-                            <h5 class="card-title font-family-roboto font-bold">{{ content.title }}</h5>
-                            <p class="card-text font-family-roboto font-regular">{{ content.description }}</p>
-                        </div>
-                    </div>
+    {% for content in contents %}
+    {% if content.section.title == section.title %}
```

Comment:
This is an example of why I suggested merging models **Section** and **SectionContent** together. I don't really understand this conditional check.